### DISCUSSION TITLE
Mermaid flowchart using chapter anchors as clickable links?

### DISCUSSION THREAD

**Discussion by ThorkellHelgason96** [community/community#103127] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have the problem in Markdown that I cannot seem to be able to reference an anchor inside a mermaid flowchart. Consider this simple example below:

(my-anchor)=
#Section I want to jump to 
Example section that I want to jump to

## Chapter with flowchart

```mermaid
graph TD;
  A[Click me](#my-anchor) --> B;
  B --> C;
  C --> D;

But this does not seem to be supported, or I cannot find ways to do this after a long search online. I know that I can use url's, but that is not what I'm aiming for as the url to my section might change with time. So an anchor like I am trying to use is more robust.

Does anyone know how to do this?

**Comment by thorimur** [comment]
In general, you'll want to use double quotes and backticks to enable Markdown, e.g. ``A["`[Click me](link)`"]`` (see [the docs](https://mermaid.js.org/syntax/flowchart.html#markdown-formatting)). However, I'm not totally sure this specific example would work; if not, linking to the relevant url followed by the markdown anchor might work (just an idea, untested). You might also want to explore the [click functionality](https://mermaid.js.org/syntax/flowchart.html#interaction) if you can.

But moreover, node formatting [currently isn't working on github anyway](https://github.com/orgs/community/discussions/106690), unfortunately. Hopefully this will be fixed soon.
